src/routes/__layout.svelte

```diff
@@ -40,7 +40,7 @@
     loadCluster().catch(() =>
       notifications.add('error', 'Unable to fetch cluster info'),
     );
-    loadSettings();
+    await loadSettings();
```

Comment:
Should all of this just be in the `load` function and not a store at all?

Reply:
One way or the other I 100% think this needs to be re-evaluated. I don't like how it works with the rest of the code and the stuff we're doing to make this work correctly in other parts of the app. 

This PR makes the cloud namespaces work so I think it's important to push it out independently of fixing that.

Reply:
Yeah thinking more on this it should definitely be hydrated in that load function and passed down to stuff and into context and we should use it there